pyVHDLParser/Token/Parser.py

```diff
@@ -354,10 +354,19 @@ def GetVHDLTokenizer(cls, iterable: Iterator[str]):
 					start.Column +=   1
 					start.Absolute += 1
 					buffer =          buffer[1:]
-					if (buffer[0] in __ALPHA_CHARACTERS__) and (buffer[1] in __ALPHA_CHARACTERS__):
-						tokenKind =     cls.TokenKind.AlphaChars
-					elif (buffer[0] in __WHITESPACE_CHARACTERS__) and (buffer[1] in __WHITESPACE_CHARACTERS__):
-						tokenKind =     cls.TokenKind.SpaceChars
+
+					if buffer[0] == "(": # Relevant for qualified expressions

```

Comment:
```suggestion
					if buffer[0] == "(":  # Relevant for qualified expressions
```
Python (styling) requires at least 2 spaces before `#`. I'm following Python coding styles loosely, but this is a good rule :).